docs/install.rst

```diff
@@ -115,13 +115,17 @@ these make much sense to you, you should definitely use the
 certbot-auto_ method, which enables you to use installer plugins
 that cover both of those hard topics.
 
-If you're still not convinced and have decided to use this method,
-from the server that the domain you're requesting a certficate for resolves
-to, `install Docker`_, then issue the following command:
+If you're still not convinced and have decided to use this method, from
+the server that the domain you're requesting a certficate for resolves
+to, `install Docker`_, then issue a command like the one found below. If
+you are using Certbot with the :ref:`Standalone` plugin, you will need
+to make the port it uses accessible from outside of the container by
+including something like ``-p 80:80`` on the command line before
+``certbot/certbot``.
 
 .. code-block:: shell
 
-   sudo docker run -it --rm -p 443:443 -p 80:80 --name certbot \
+   sudo docker run -it --rm --name certbot \
```

Comment:
why remove `443`?

Reply:
Because `tls-sni` is dead.

I also tried to be a little general about the ports needed which I think is fine due to all of the warnings about using Certbot in Docker in the beginning of this section.

Reply:
but it's not dead for existing users

Reply:
I don't think existing users are reading how to set it up again though. I can include "or ``-p 443:443``" in the sentence above this if you prefer.

Reply:
yeah that sounds good

Reply:
done